### DISCUSSION TITLE
new quick search is nice, but please make that optional

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#12384] [Dashboard]
https://github.com/home-assistant/frontend/pull/12324

the new Magnifier icon on desktop is not that interfering, but especially mobile, the  takes a way costly screen estate we use for our tabs.

since searching would not be for anyone, and probably mostly when developing, it would be great if we wouldn't have this in the menu bar by default, and make it optional

Hiding it via card-mod-theme is probably possible, doing something like this myself currently

```
        ha-icon-button {
          color: {{'transparent' if not is_state('input_select.mode','Developer')}};
        }
```

can also use `display: none` , but that disorders the rest of the menu items and causes them to be displaced....

https://community.home-assistant.io/t/card-mod-super-charge-your-themes/212176/1079

**Comment by matthiasdebaat** [comment]
We're looking into having a single search in the sidebar, instead of magnify icons on different pages.

**Comment by Mariusthvdb** [comment]
thanks for moving the [search to the overflow](https://github.com/home-assistant/frontend/pull/12552) three dots Menu button on Mobile.  That's a huge relieve for the already crammed menubar.

Please also consider moving the search on desktop? it would still allow any user in a default setting to use it. It would however also enable people that dont want it, to hide it from that menu button (using card-mod)

Ofc, a core toggle to have it displayed on admin only would still be preferable imho, but if thats a bridge too far, moving it to the menu button  would be a very nice solution too.

ofc, the pulldown overflow Paulus mentioned in the #beta for 2022.5 would be way cool. If that is being worked on, please also see to a configuration option to not enable it for non-admin at least

**Reply by Mariusthvdb** [reply]
edit: wrong place. moved to the issue at  https://github.com/home-assistant/frontend/issues/12659
